doc/overview.rst

```diff
@@ -830,8 +813,78 @@ tune the model for your use case with your custom dataset. The following assumes
 familiarity with `SageMaker training jobs and their
 architecture <https://docs.aws.amazon.com/sagemaker/latest/dg/how-it-works-training.html>`__.
 
-Fine-tune a Pre-trained Model on a Custom Dataset
--------------------------------------------------
+Low-code fine-tuning with the JumpStartEstimator class
+------------------------------------------------------
+
+You can fine-tune a built-in algorithm or pre-trained model in just a few lines of code using the SageMaker Python SDK.
+
+First, find the model ID for the model of your choice in the :doc:`Built-in Algorithms with pre-trained Model Table<./doc_utils/pretrainedmodels>`.
+
+Using the model ID, define your training job as a JumpStart estimator. Run ``estimator.fit()`` on your model, pointing to the training data to use for fine-tuning.
+Then, use the ``deploy`` method to automatically deploy your model for inference. In this example, we use the GPT-J 6B model from HuggingFace.
+
+.. code:: python
+
+    from sagemaker.jumpstart.estimator import JumpStartEstimator
+
+    model_id = "huggingface-textgeneration1-gpt-j-6b"
+    estimator = JumpStartEstimator(model_id=model_id)
+    estimator.fit(
+        {"train": training_dataset_s3_path, "validation": validation_dataset_s3_path}, logs=True
```

Comment:
Could you confirm whether `logs=True` is an acceptable value?

It seems that it must be a list of `str`: see [code](https://github.com/aws/sagemaker-python-sdk/blob/6a252f9ab16febdddb0c7b691677e6c0db41f826/src/sagemaker/jumpstart/estimator.py#L607).